crates/biome_cli/tests/snapshots/main_cases_rules_via_dependencies/doesnt_enable_rules_when_recommended_is_false.snap

```diff
@@ -0,0 +1,47 @@
+---
+source: crates/biome_cli/tests/snap_test.rs
+expression: redactor(content)
+---
+## `biome.json`
+
+```json
+{
+  "linter": {
+    "rules": {
+      "recommended": false
```

Comment:
Should we add an example where we turned off this global `recommended` and set to `recommended` a domain? This is to ensure that the setting of the domain overrides the global one for the rules of the domain.
Perhaps the test already exists.

Reply:
Will do!